src/components/molecules/NavList.tsx

```diff
@@ -1,64 +1,66 @@
 import { List } from '@mui/material'
 import { useNavigate } from 'react-router-dom'
-import { SubheaderLinks } from './subheaderLinks/SubheaderLinks'
-import {
-  Genre,
-  Role,
-  genres,
-  navLinks,
-  roles,
-} from '../../common/constants/enums'
+import { navLinks, genres, roles } from '../../common/constants/enums'
 import { enumMappings } from '../../common/constants/constants'
 import { NavLinks } from './navLinks/NavLinks'
 import { NavStyle } from '../../common/utils/newStyle'
 
 type ListButtonProps = {
   label: string
-  filter: string
+  filter?: string
   path: string
 }
-type SubheaderLinksProps = {
-  items: readonly (Genre | Role)[]
-  navigate: (route: string) => void
-}
+
+const orderedLinks: ((typeof genres)[number] | (typeof roles)[number])[] = [
+  'animation',
+  'documentary',
+  'director_of_photography',
+  'director_producer',
+  'series',
+]
 
 export function NavList() {
   const navigate = useNavigate()
 
-  const generateButtons = (
-    items: readonly (Genre | Role)[]
-  ): ListButtonProps[] =>
-    items.map(item => ({
-      label: enumMappings[item].label,
-      filter: item,
-      path: enumMappings[item].url, 
-    }));
+  const combinedLinks: ListButtonProps[] = navLinks.map(item => ({
+    label: item.label,
+    path: item.route,
+  }))
```

Comment:
🔧 Similar question here. Why convert the `navlinks` to `ListButtonProps` here and then just convert them to `NavLinks` later? Why not just go straight to `NavLinks`?

Reply:
You're right. 
Hate when I don't catch stuff like this.